services/scheduler/src/index.js

```diff
@@ -0,0 +1,57 @@
+const defaultLogger = require('./logger');
+
+class Scheduler {
+    constructor(config, flowsDao, schedulePublisher) {
+        this._logger = defaultLogger;
+        this._config = config;
+        this._flowsDao = flowsDao;
+        this._schedulePublisher = schedulePublisher;
+    }
+
+    async _scheduleFlows() {
+        const flows = await this._flowsDao.findForScheduling({
+            limit: parseInt(this._config.get('POLLING_TASK_COUNT'))
```

Comment:
this limit could be a part of an implementation, not a framework. How do you think, @anton-kotenko ?